actor/src/main/scala/org/apache/pekko/actor/LightArrayRevolverScheduler.scala

```diff
@@ -200,6 +202,11 @@ class LightArrayRevolverScheduler(config: Config, log: LoggingAdapter, threadFac
         s"Task scheduled with [${delayNanos.nanos.toSeconds}] seconds delay, " +
         s"which is too far in future, maximum delay is [${(tickNanos * Int.MaxValue).nanos.toSeconds - 1}] seconds")
 
+  private def checkZeroPeriod(delayNanos: Long): Unit =
+    if (delayNanos <= 0)
+      throw new IllegalArgumentException(
+        "Task scheduled with zero or negative period, which is create an an infinite loop")
```

Comment:
use:
- `delay :xxx expected > 0` or `delay should > 0`.

user do not know what's `delayNanos`

Reply:
what about "Task scheduled with [${delay.toSeconds}] seconds delay, which means creating an infinite loop. The expected delay must be greater than 0."?